Marlin/Marlin_main.cpp

```diff
@@ -2962,6 +2990,23 @@ inline void gcode_M17() {
   enable_all_steppers();
 }
 
+#ifdef RESUME_FEATURE
+  inline void gcode_M19() {
+    if(code_seen('Z'))
+    {
+      gcode_get_destination(); // For Z
+      prepare_move();
```

Comment:
Yes, there's certainly no problem in having it there, so better be safe than sorry.